src/components/sign-in/sign-in.module.scss

```diff
@@ -3,27 +3,34 @@
 
 .accountDropdown {
   margin-top: 20px;
+  background: linear-gradient(180deg, rgba(250, 250, 250, 0.7) 1.5%, rgba(227, 234, 255, 0) 103.92%), #e3eaff;
```

Comment:
Looks like the button background should to be applied only when the dropdown is expanded

| Figma | Implementation |
|--------|--------|
| <img width="234" alt="image" src="https://github.com/user-attachments/assets/aba55122-3dd2-43bf-a819-43009274dd14"> | <img width="234" alt="image" src="https://github.com/user-attachments/assets/7a7d9500-4227-4f69-8992-cf165ddde7b3"> |

Reply:
oh ofc, thanks 👍